### DISCUSSION TITLE
ReadMe and Install Guide Reccomendations + A Menuing Request

### DISCUSSION THREAD

**Discussion by TheSwampCoven** [profi200/open_agb_firm#126] [Ideas]
Hi! Firstly, love this software so much. Easily the best way to play GBA games barring using an actual GBA.

I saw the border feature was added and it took me all morning to find the instructions. Might be best to include instructions on how to install one near the general install instructions and/or the ReadMe.

Also, [Open AGB Launcher](https://gbatemp.net/download/open-agb-launcher.36828/) is a super useful little cia that I thought you might wanna shout out somewhere below the feature list or in the ReadMe. It still works in 2023 with the latest versions of fastboot, godmode, and luma! (Though the incompatibility with b9s is frustrating.) I found it very useful what with my adhd and my constant forgetting to input my keybind to boot to agb firm.

One final thing, I know a more elaborate UI is probably well down the road, but have you thought of making it so you can hold down the dpad to scroll through menu options faster? Not even sure if its possible, but I have a lot of games on my SD and sometimes my carpal tunnel can't handle it lol.

Again, love this project.

**Comment by profi200** [comment]
Yes, i didn't really add instructions about border conversion yet. I will see where it fits in the README later.

Not sure on the way to launch oaf from HOME menu. I'm not sure if i should change my mind to make oaf capable to work as AGB_FIRM replacement. I would have to implement quite a bit of stuff to make this work. It doesn't work with b9s because it can't launch preloaded FIRM files from RAM.

I'm experimenting with GUI stuff right now. One library i tested is simply too big. So it's custom GUI again. And that is a pain in the ass as well needing to design the whole thing from scratch.
